actors/verifreg/src/lib.rs

```diff
@@ -428,6 +426,7 @@ where
     Ok(x.0 as TokenAmount)
 }
 
+// Invokes Mint on a datacap token actor.
```

Comment:
These three methods use TokenAmount types but I would have expected DataCap

Reply:
I guess I could go either way. The callers convert immediately before the call. The verifreg "knows" its a token.